tests/selfapps_layered_e2e.ps1

```
@@ -0,0 +1,342 @@
+            $psi.FileName = $exePath
+            $psi.UseShellExecute = $false
+            $psi.RedirectStandardOutput = $true
+            $psi.RedirectStandardError = $true
+            $proc = [System.Diagnostics.Process]::Start($psi)
+            $stdoutTask = $proc.StandardOutput.ReadToEndAsync()
+            $stderrTask = $proc.StandardError.ReadToEndAsync()
+            $exeTimeoutMs = 60000
+            if ($proc.WaitForExit($exeTimeoutMs)) {
+                $exeExit = $proc.ExitCode
+            } else {
+                try { $proc.Kill() } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)